python/pyarrow/_parquet.pyx

```diff
@@ -1830,7 +1830,9 @@ cdef shared_ptr[WriterProperties] _create_writer_properties(
         dictionary_pagesize_limit=None,
         write_page_index=False,
         write_page_checksum=False,
-        sorting_columns=None) except *:
+        sorting_columns=None,
+        store_decimal_as_integer=False) except *:
+
```

Comment:
Unnecessary empty line
```suggestion
```

Reply:
I believe this is resolved in the latest commit